src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -68,10 +74,23 @@ trait ExecutionComponent {
     } yield execution.status)
 
   // see workflowExecutionsByStatuses
-  def executionStatusesByWorkflowExecutionIdAndCallFqns(workflowExecutionId: Int, callFqns: Traversable[String]) = {
-    for {
+  def executionStatusesByWorkflowExecutionIdAndCallFqns(workflowExecutionId: Int, scopeKeys: Traversable[ExecutionDatabaseKey]) = {
```

Comment:
method should probably be renamed to `executionStatusesByWorkflowExecutionIdAndCallKey` or something like that